sdk/keyvault/tests.yml

```diff
@@ -10,11 +10,13 @@ resources:
 jobs:
 - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
-    MaxParallel: 1
     ServiceDirectory: keyvault
     EnvVars:
-      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```

Comment:
Once these changes go through can you please file a follow-up item to remove these secrets in keyvault and clean-up the associated resources in our test sub.

Reply:
I opened Azure/azure-sdk-for-net#9170 to track.